corehq/sql_accessors/sql_templates/get_latest_ledger_transaction.sql

```diff
@@ -7,6 +7,6 @@ BEGIN
     RETURN QUERY
     SELECT * FROM form_processor_ledgertransaction
     WHERE case_id = p_case_id AND section_id = p_section_id AND entry_id = p_entry_id
-    ORDER BY report_date DESC limit 1;
+    ORDER BY report_date DESC, "id" DESC limit 1;
```

Comment:
If we want this to get updated in prod (which I assume we do) I think you need to rename this to a new file (e.g. `get_latest_ledger_transaction_2.sql` and then add a migration that executes this. you should also replace previous references to this file with `noop_migration()` so that they don't needlessly run. at least I'm pretty sure that's what @snopoke does